processor/src/chiplets/hasher/tests.rs

```diff
@@ -607,6 +607,8 @@ fn check_hasher_state_trace(trace: &[Vec<Felt>], row_idx: usize, init_state: Has
     }
 }
 
+// TODO(serge): Convert args to Step? Trait unstable:
```

Comment:
Thoughts on whether we want this to depend on the nightly cfg? If not, should we leave a cleaned up version of this comment about waiting on stable?

Reply:
Same todo comes up a few times

Reply:
This only comes up in tests, so I think it's fine as is